app/controllers/container_topology_controller.rb

```diff
@@ -22,6 +22,11 @@ def data
     render :json => {:data => generate_topology(params[:id])}
   end
 
+  def refresh_provider
+    render :nothing => true
+    EmsRefresh.queue_refresh([params[:id]])
```

Comment:
@chessbyte can you perhaps have a look too? https://github.com/ManageIQ/manageiq/pull/5991/files#r49301654